src/Components/TextInput/TextInput.jsx

```diff
@@ -12,19 +12,19 @@ const TextInput = ({
   autoComplete = null,
   ...props
 }) => {
-  const [value, setValue] = useState("");
+  const [val, setVal] = useState("");
```

Comment:
why you renamed this?

Reply:
i removed this because we will have value in the component.